packages/appstore/src/components/currency-switcher-card/real/__tests__/real-account-switcher.spec.tsx

```diff
@@ -40,9 +40,11 @@ describe('RealAccountSwitcher', () => {
     it('should render AccountNeedsVerification component with the correct pending status', () => {
         const mock = mockStore({
             traders_hub: {
-                multipliers_account_status: 'pending',
                 is_eu_user: true,
             },
+            client: {
+                mf_account_status: 'pending',
```

Comment:
`mf_account_status` is being used in many places and the status options are being hard-coded, can we look for an alternative way of not hard-coding the values!